### DISCUSSION TITLE
Respawn Shuttle despawn timer problems

### DISCUSSION THREAD

**Discussion by luisech4** [FakeFishGames/Barotrauma#10114] [Ideas and suggestions]
Copying my message from #baro-critique-and-ideas on the discord

My friends and I had to restart our mission because we were completely blocked in by our respawn shuttle, we tried everything we could think of for around 10 minutes and hoped that it would eventually despawn. After looking around the [wiki](https://barotraumagame.com/wiki/Serversettings.xml#:~:text=Sets%20maximum%20time,the%20first%20outpost.) it seems that respawn shuttles don't despawn once they dock with your sub even if you undock? 

I'm not sure if this was a bug or just an overlooked feature, but think it'd be a good idea for the despawn timer to be extended for a couple more minutes once it docks rather than causing it to stay indefinitely or for the despawn timer to kick back in after the shuttle undocks. I forgot to record or take pictures in game but I have provided an illustration of our situation here:
![rockandahardplace](https://user-images.githubusercontent.com/116126469/196563178-dc23bcba-a6c3-4504-9584-ffce975b0ea9.png)

**Comment by mygamingaccount** [comment]
![skill_issue](https://user-images.githubusercontent.com/68995233/196568195-a3f62d28-54c1-41e6-9e59-de1ac1c43274.png)

**Comment by gerakis** [comment]
It should despawn 5 minutes after spawning, regardless if it docked with anything. If it undocks and there is no one on board it should despawn almost instantly, like within 30 seconds.

**Comment by Ek-Videogames** [comment]
Yeah respawn shuttle is inconsistent, the time until shuttle despawns entirely after "returning" should be alot shorter & probably should display a timer in the top-left if it's a minute or longer the same way every other timer does.
